lib/ueberauth/strategy/google.ex

```diff
@@ -76,6 +76,7 @@ defmodule Ueberauth.Strategy.Google do
       expires: !!token.expires_at,
       expires_at: token.expires_at,
       scopes: scopes,
+      token_type: token.token_type,
```

Comment:
@tsubery might this be better as `token[:token_type]` to avoid exceptions if it is missing?  We had a similar issue with `ueberauth_slack`: https://github.com/ueberauth/ueberauth_slack/pull/10